addon-test-support/fire-event.js

```diff
@@ -4,6 +4,7 @@ const { merge } = Ember;
 const DEFAULT_EVENT_OPTIONS = { bubbles: true, cancelable: true };
 const KEYBOARD_EVENT_TYPES = ['keydown', 'keypress', 'keyup'];
 const MOUSE_EVENT_TYPES = ['click', 'mousedown', 'mouseup', 'dblclick', 'mouseenter', 'mouseleave', 'mousemove', 'mouseout', 'mouseover'];
+const FILE_EVENT_TYPES = ['change'];
```

Comment:
I don't consider `change` to be an event for files only. I think that, making an exception, I'd colocate the logic to generate this special event along with the helper, as I don't think that it has any utility elsewhere.